---
title: AddCell presented at the CRISPR as a research tool in cancer and regenerative medicine.

event: [ELIXIR-DE-Hackathons](https://www.scilifelab.se/event/crispr-as-a-research-tool-scilifelab-international-symposium/)
event_url: https://www.scilifelab.se/event/crispr-as-a-research-tool-scilifelab-international-symposium/

location: Karolinska Institutet, Sweden
# address:
#   street: 450 Serra Mall
#   city: Stanford
#   region: CA
#   postcode: '94305'
#   country: United States

summary: Advancing spatially resolved transcriptomics demands community-driven benchmarking, spotlighted by Nature's call to action using OpenEBench and OMNIBENCHMARK platforms for tool evaluation and dataset curation.
abstract: Spatially resolved transcriptomics enables the examination of gene expression within the natural tissue setting. We're witnessing a surge in tool development for subsequent analyses, alongside the rise of datasets commonly used to assess new tool quality. Despite this, there's a gap in a community-led, ongoing assessment of spatially resolved transcriptomics tools. A recent Nature article underscores the importance of remedying this through benchmarking facilitated by OpenEBench and OMNIBENCHMARK platforms. These platforms offer a structure for organizing computational tools and reference datasets, aiming for adaptability as new computational methods arise.

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: '2023-12-10T13:00:00Z'
date_end: '2023-12-15T13:00:00Z'
all_day: true

# Schedule page publish date (NOT talk date).
publishDate: '2017-01-01T00:00:00Z'

authors: []
tags: []

# Is this a featured talk? (true/false)
featured: false

image:
  caption: ''
  focal_point: Right

# links:
#   - icon: twitter
#     icon_pack: fab
#     name: Follow
#     url: https://twitter.com/georgecushen
# url_code: ''
# url_pdf: ''
# url_slides: ''
# url_video: ''

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
# projects:
#   - example
---

<!-- {{% callout note %}}
Click on the **Slides** button above to view the built-in slides feature.
{{% /callout %}}

Slides can be added in a few ways:

- **Create** slides using Hugo Blox Builder's [_Slides_](https://docs.hugoblox.com/reference/content-types/) feature and link using `slides` parameter in the front matter of the talk file
- **Upload** an existing slide deck to `static/` and link using `url_slides` parameter in the front matter of the talk file
- **Embed** your slides (e.g. Google Slides) or presentation video on this page using [shortcodes](https://docs.hugoblox.com/reference/markdown/).

Further event details, including [page elements](https://docs.hugoblox.com/reference/markdown/) such as image galleries, can be added to the body of this page. -->